bedrock/firefox/templates/firefox/new/sem/unsupported-browser/scene1.html

```diff
@@ -0,0 +1,22 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/sem/base.html" %}
+
+{% block optimizely %}
+  {% if switch('firefox-new-unsupported-browser-optimizely', ['en-US']) %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+
+{% block head_content %}
+  <h2 id="head_content_title" data-experience="unsupported-browser">
+    Smooth and Reliable
+  </h2>
+
+  <p>
+    Firefox stays compatible with your favorite sites -- Google, YouTube, Facebook, Amazon – so you always browse fast, reliable and smooth.
```

Comment:
Should the dashes here be `&mdash;`'s? Maybe at least make them consistent - the first one's a double hyphen? Is that on purpose?